RELEASE_NOTES.md

```diff
@@ -1,5 +1,152 @@
 # Release Notes
 
+## 2.2.0
+### Features
+- Experimental support for shortname aliasing has been added. This is not enabled by default, but can be turned on by setting the environment variable `CONTAINERS_SHORT_NAME_ALIASING` to `on`.
```

Comment:
I'm not sure where, but it would be nice to have a pointer to more info on this.

Reply:
We can update once the blog post is released.

Reply:
Or even better. Point to the upstream docs: https://github.com/containers/image/blob/master/docs/containers-registries.conf.5.md#short-name-aliasing